src/plugins/opensearch_dashboards_overview/public/application.tsx

```diff
@@ -65,6 +65,7 @@ export const renderApp = (
           newsfeed$={newsfeed$}
           solutions={solutions}
           features={features}
+          branding={core.injectedMetadata.getBranding()}
```

Comment:
nit: can we put this into a constant defined above to match the format

Reply:
yes thx!